### ISSUE TITLE
Can't find the correct way to download SSL enabled Quickfix package for Windows.

### ISSUE THREAD

**Issue by emirhanpaksoy** [quickfix/quickfix#605]
Hello,

I've been trying to create a client using QuickFIX/Python for a few weeks now. Since I'm working on Windows, I initially faced quite a few challenges with the installation. Eventually, while searching through carious issues, I came accross this link: https://download.lfd.uci.edu/pythonlibs/archived/quickfix-1.15.1-cp39-cp39-win_amd64.whl .
I downloaded the corresponding Python version and succesfully set up the client. However, I discovered that SSL connections couldn't be established with this version of quickfix. After reviewing the merges, I noticed that SSL-Python related improvements habe been made in the latest version, but it seems the .whl package I found is an older version.

In short, I'm trying to create an environment on Windows where I can enable SSL using Python. If you have a .whl for this, I would appreciate it if you could share it. I also tried building from scratch using CMake but wasn't very succesful. I would greatly appreciate any assistance you can provide. If anyone has succesfully set up a FIX engine with SSL using Python on Windows before, I would be grateful if you could share your insights.

**Comment by pepe454** [comment]
@emirhanpaksoy  One approach you can try is using stunnel - https://www.stunnel.org/

In the stunnel config you can set up a tunnel like this: 

[tunnel-name]
debug = 6
verify = 2
client = yes
accept = 9000
connection = target-ip-address:port 
cert = /your/cert/path
CAfile = /your/cafile/path

In your quickfix config, a session can connect to a local ssl tunnel like so: 

[SESSION]
SocketConnectHost=localhost
SocketConnectPort=12000
... more config follows

This way your quickfix engine doesnt directly use SSL but all communication from the engine gets sent through the tunnel to your FIX counterparty. However, you have to configure stunnel and run it as a service (works on linux and windows).
